docker/install

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+set -eux
+
+echo "*** Downloading Schema Registry"
+# download confluent binaries
+curl -sSL http://packages.confluent.io/archive/5.5/confluent-community-$CONFLUENT_VERSION-$SCALA_VERSION.tar.gz | tar xz \
+confluent-$CONFLUENT_VERSION/bin/schema-registry-run-class \
+confluent-$CONFLUENT_VERSION/etc/schema-registry \
+confluent-$CONFLUENT_VERSION/share/java/confluent-common \
+confluent-$CONFLUENT_VERSION/share/java/rest-utils \
+confluent-$CONFLUENT_VERSION/share/java/schema-registry
+
+mv confluent-$CONFLUENT_VERSION/* .
+
+# Remove bash as our images doesn't have it, and it isn't required
+sed -i 's~#!/bin/bash~#!/bin/sh~g' bin/*
+# Remove bash syntax relating to irrelevant CYGWIN (prevents console errors)
```

Comment:
@JBAhire @kotharironak @laxmanchekka @ravisingal I think we have some other images that will be able to use this to kill cygwin errors as many people copy/paste the same basic start scripts